### ISSUE TITLE
manager: cannot find stage  error="<nil>"

### ISSUE THREAD

**Issue by frebib** [drone/drone#2832]
I'm playing with the webhook functionality. I noticed this in the drone-server logs
```
WARN[0179] manager: cannot find stage                    error="<nil>" step-id=46
```

I feel like this code is wrong, and should be `err != nil`
https://github.com/drone/drone/blob/master/operator/manager/manager.go#L468

**Comment by tboerger** [comment]
You are right, that check seems to be wrong

**Comment by bradrydzewski** [comment]
good catch, closed by 807738d5970647c564ff71f0dbd19c7ca080e4da
